apps/cra-template-mockbee-social-media/template/src/backend/controllers/CommentsController.js

```diff
@@ -0,0 +1,280 @@
+import { Response } from "miragejs";
+import { formatDate, requiresAuth } from "../utils/authUtils";
+import { v4 as uuid } from "uuid";
+
+/**
+ * All the routes related to post comments are present here.
+ * */
+
+/**
+ * This handler handles getting all comments for a particular post in the db.
+ * send GET Request at /api/comments/:postId
+ * */
+
+export const getPostCommentsHandler = function (schema, request) {
+  const postId = request.params.postId;
```

Comment:
This could also be written using object de-structure as `const { postId } = request.params;`